test/tools/ossfuzz/yulProto.proto

```diff
@@ -58,23 +58,14 @@ message Create {
 }
 
 message FunctionCall {
-  enum Returns {
-    ZERO = 1;
-    SINGLE = 2;
-    MULTIDECL = 3;
-    MULTIASSIGN = 4;
-  }
-  required Returns ret = 1;
-  // Indexes an existing function
-  required uint32 func_index = 2;
-  required Expression in_param1 = 3;
-  required Expression in_param2 = 4;
-  required Expression in_param3 = 5;
-  required Expression in_param4 = 6;
-  required VarRef out_param1 = 7;
-  required VarRef out_param2 = 8;
-  required VarRef out_param3 = 9;
-  required VarRef out_param4 = 10;
+  required Expression in_param1 = 1;
+  required Expression in_param2 = 2;
+  required Expression in_param3 = 3;
+  required Expression in_param4 = 4;
+  required VarRef out_param1 = 5;
+  required VarRef out_param2 = 6;
+  required VarRef out_param3 = 7;
+  required VarRef out_param4 = 8;
```

Comment:
The change here is that `Returns` and function index have been removed. In and out parameters remain but with different offsets within the message (unimportant detail).